pvacseq/lib/generate_fasta.py

```diff
@@ -112,7 +113,10 @@ def main(args_input = sys.argv[1:]):
 
         if variant_type == 'FS':
             wildtype_subsequence, mutant_subsequence = get_frameshift_subsequences(position, full_wildtype_sequence, peptide_sequence_length, line)
-            mutant_subsequence += line['downstream_amino_acid_sequence']
+            downstream_sequence = line['downstream_amino_acid_sequence']
+            if args.downstream_sequence_length and len(downstream_sequence) > int(args.downstream_sequence_length):
```

Comment:
I ended up making `downstream_sequence_length` a `type=int` in generate_fasta. This required some changes in the pipeline as to how the parameter is passed in but it removes the need to for the explicit conversion. Since in generate_fasta only int values are valid for this parameter anyway I think this is what I should've done from the beginning.